### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CSVPreviewData_ScreenReader: JAWS is not reading the heading level for table heading.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6072]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #MAS2.4.6; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:*
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
JAWS version : 2019.1810.64 ILM

**Additional Details:**
Tool Name: JAWS
MAS Reference:MAS1.3.1,MAS2.4.6

**Steps to Reproduce:**

1. Install the app Azure Data Studio.
2. Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button.
3. Start JAWS.
4. Navigate till "Steps 2" using up and down arrows .
5. Observe that screen reader is not announcing heading level for "Steps 2" and table heading.

**Actual:**
**Issue 1 :**
Screen reader only announce "Steps 2 " for heading level and not announcing heading level for it.

**Issue 2 :**
Screen reader only announce"This operation analysed the input file structure to generate the preview below for upto the first 50 rows" for table heading.

**Similar issue is also observed with NVDA.**
**Similar issue is also observed in following pages :**
**1. CSVModifyColumns :** Connect   to database->navigate to the Data services folder->HDFS Folder->   navigate to CSV files folder->Right click on the files->Select option   Create external table from CSV->Click on the "Next"   button->Click on next button.
**2.CSVSummary:** Connect   to database->navigate to the Data services folder->HDFS Folder->   navigate to CSV files folder->Right click on the files->Select option   Create external table from CSV->Click on the "Next"   button->click on next button.
**3.CSVSelectDestinationDatbse:** Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV files

**Expected:**
**Issue 1 :**
Screen reader should announce heading level for  " Steps 2 " 
For example, " heading level 1  Steps 2 " 

**Issue 2 :**
Screen reader should announce heading level for " This operation analysed the input file structure to generate the preview below for upto the first 50 rows" for table heading.
For example,  "heading level 2 This operation analysed the input file structure to generate the preview below for upto the first 50 rows ".

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User with visual and motor disabilities will find difficult to differentiate between header and the rest text and the purpose for the page

**MAS Reference:
MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**MAS2.4.6-**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}](url)

**Attachment for Reference:**
[6072_[Accessibility]A11y_AzureDataStudio_CSVPreviewData_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3300938/6072_.Accessibility.A11y_AzureDataStudio_CSVPreviewData_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes
